config/lpdc-management/content/form.ttl

```diff
@@ -1102,6 +1105,114 @@ ext:contactpointsPg a form:PropertyGroup;
         form:displayType displayTypes:defaultInput ;
         sh:group ext:contactpointsPg-sub .
 
+        ##########################################################
+        #  property-group: contactPointAddress
+        ##########################################################
+        ext:contactPointAddressPg a form:PropertyGroup;
+            sh:name "Adres" ;
+            sh:order 1 .
+        
+        ##########################################################
+        #  listing-scope: contactPointAddress
+        ##########################################################
+        ext:contactPointAddressS a form:Scope;
+          sh:path lpdcExt:address.
+        
+        ##########################################################
+        #  Listing: contactPointAddress
+        ##########################################################
+        ext:contactPointAddressL a form:Listing;
+          form:each ext:contactPointAddressSf;
+          form:scope ext:contactPointAddressS;
+          form:createGenerator ext:contactPointAddressG;
+          sh:group ext:contactPointAddressPg;
+          form:addLabel "Voeg adres toe";
+          form:canAdd true;
+          form:canRemove true;
+          sh:maxCount 1;
+          sh:order 1 .
+                 
+        ##########################################################
+        #  Generator: contactPointAddress
+        ##########################################################
+        ext:contactPointAddressG a form:Generator;
+          form:prototype [
+            form:shape [
+              a locn:Address ;
+              adres:volledigAdres " "@nl;
+              adres:postcode " ";
+              adres:gemeentenaam " "@nl
+            ];
+          ];
+          form:dataGenerator form:addMuUuid.
+        
+        ##########################################################
+        #  property-group: contactPointAddress-sub
+        ##########################################################
+        ext:contactPointAddressPg-sub a form:PropertyGroup;
+            sh:order 1 .
+        
+        ##########################################################
+        #  Subform: contactPointAddress
+        ##########################################################
+        ext:contactPointAddressSf a form:SubForm;
+          form:removeLabel "Verwijder adres";
+          form:includes ext:contactPointAddressFullAddressF;
+          form:includes ext:contactPointAddressPostcodeF;
+          form:includes ext:contactPointAddressMunicipalityF.
+
+          ##########################################################
+          # Field: full address
+          ##########################################################
+          ext:contactPointAddressFullAddressF a form:Field ;
+            sh:name "Adres" ;
+            sh:order 1 ;
+            form:language "nl";
+            form:validations [
+              a form:RequiredConstraint ;
```

Comment:
I'm not sure if the fields are required.

Reply:
none it seems see `0..1` in the model